### PR TITLE
add team list view

### PR DESCRIPTION
**Pull request by FernandoArevaloE**
* listByLeaderId service fixed

### CODE REVIEW

File: web/src/views/TeamList.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            }
+            await axios.get(`http://localhost:4000/v1/team?actorId=${actorId}`,
+                {
+                    headers: headers
+                }
+            )
+            .then(response => setTeamList(response.data.data.length > 0 ? response.data.data : null))
+            .catch(error => {
+                if (error.response.status === 404) {
+                    setTeamList(null);
```
Comment (andymotagl) [inline web/src/views/TeamList.js:42]: SetTeamList should be added to hook parameters

File: app/src/service/team.js [hunk 2]
```diff
@@ -9,8 +9,8 @@ const { BadRequest, NotFound, NotModified } = require('../web/apiResponse');
 
 const listByLeaderId = async (actorId) => {
     if (!actorId || !uuidTool.validate(actorId)) throw new BadRequest('Invalid Actor Id');
-    const result = await teamDAO.getTeamByLeaderId(actorId);
-    if (!result.length) throw new NotFound();
+    const result = await teamDAO.listByLeaderId(actorId);
+    if (result.length === 0) throw new NotFound();
```
Comment (andymotagl) [inline app/src/service/team.js:13]: You miss to update the documentation

File: web/src/views/NewTeam.js [hunk 3]
```diff
@@ -26,7 +26,6 @@ const NewTeam = () => {
     const handleInput  = (event) => {
         setTeam({
             ...team,
-            actorId: actor.actor.actor_id,
```
[the remark this answers is not in the document]
Comment (FernandoArevaloE) [reply web/src/views/NewTeam.js:29]: done

File: web/src/views/NewTeam.js [hunk 4]
```diff
@@ -39,7 +38,8 @@ const NewTeam = () => {
             await axios.post('http://localhost:4000/v1/team/', team, {
                 headers: {
                     'Content-Type': 'application/json',
-                    'Authorization': `Bearer ${token}`
+                    'Authorization': `Bearer ${token}`, 
+                    'actor': actor.actor.actor_id
```
[the remark this answers is not in the document]
Comment (FernandoArevaloE) [reply web/src/views/NewTeam.js:42]: done

File: web/src/views/TeamList.js (hunk 1 above)
Comment (FernandoArevaloE) [reply web/src/views/TeamList.js:42]: Really? I'm note getting any warning or error hehe

**Review by andymotagl — APPROVED** [review]

**Pull request merged**